lib/futurism/channel.rb

```diff
@@ -37,11 +38,17 @@ def receive(data)
     def resource(signed_params:, sgid:)
       return GlobalID::Locator.locate_signed(sgid) if sgid.present?
 
-      Rails
-        .application
-        .message_verifier("futurism")
+      message_verifier
         .verify(signed_params)
```

Comment:
The message_verifier was extracted to its own module to centralize the instantiation of the message verification class